### ISSUE TITLE
Could we detect clips with image recognition?

### ISSUE THREAD

**Issue by RusseII** [pillargg/pillar_algos#32]
For example, when we see the below image we know that a game just ended. There is almost always a good clip right before this event. 
![image](https://user-images.githubusercontent.com/15036618/115015779-d06d7d80-9e79-11eb-9ced-d11d1617d4e6.png)

This is outside of scope for the current implementation, but I wanted to note this possible extension down.

**Comment by pomkos** [comment]
Yes but I stopped exploring this avenue cuz @gatesyp indicated it is out of our current budget? I'd be happy investigate again though.

**Comment by chand1012** [comment]
> @gatesyp indicated it is out of our current budget.

What if we took the video, converted it into a really low frame rate (like 2-3 frames per second) and used an ML model to compare the static images? Could that bring the cost down?
